SKILL.tsv

```diff
@@ -1089,35 +1089,35 @@ SKILL_20150317_001088	Fire Ball
 SKILL_20150317_001096	Fire Property attack + #{CaptionRatio}#{nl} Additional Damage + #{CaptionRatio2}#{nl}Duration #{CaptionTime}# seconds
 SKILL_20150317_001097	Flare
-SKILL_20150317_001098	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Explode enemy in abnornal state except petrified and frozen state.
+SKILL_20150317_001098	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Inflict damage by exploding the enemy affected by a status effect. Excluding both petrified and frozen status effects.
 SKILL_20150317_001099	Attack #{SkillAtkAdd}#
 SKILL_20150317_001100	Fire Pillar
 SKILL_20150317_001101	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl}Summon fire pillar to damage enemies.
 SKILL_20150317_001102	Attack #{SkillAtkAdd}#{nl}Duration #{CaptionTime}# seconds
 SKILL_20150317_001103	Hell Breath
-SKILL_20150317_001104	{#993399}{ol}[Magic] - [Fire]{/}{/}{nlContinuously fume flame forward.
+SKILL_20150317_001104	{#993399}{ol}[Magic] - [Fire]{/}{/}{nl} Continuously burn enemies with a cone of flame.
 SKILL_20150317_001105	Attack #{SkillAtkAdd}#{nl}Consume #{CaptionRatio}#% SP per 0.5 seconds
 SKILL_20150317_001106	Ice Bolt
-SKILL_20150317_001107	{#993399}{ol}[Magic] - [Ice] - [Stab]{/}{/}{nl}Blow Ice force with chance of freezing enemy.
+SKILL_20150317_001107	{#993399}{ol}[Magic] - [Ice] - [Stab]{/}{/}{nl} Inflict damage to a target by shooting a bolt of ice. There is a chance to freeze the target.
 SKILL_20150317_001108	Attack #{SkillAtkAdd}#{nl}{#339999}{ol}[Frozen]{/}{/} Duration 3 seconds
 SKILL_20150317_001109	Ice Pike
-SKILL_20150317_001110	{#993399}{ol}[Magic] - [Ice] - [Stab] {/} {/}} {nl}Create ice waves with chance of freezing enemy.
+SKILL_20150317_001110	{#993399}{ol}[Magic] - [Ice] - [Stab] {/} {/}} {nl} Inflict damage to a target by causing a wave of ice. There is a chance to freeze the target.
 SKILL_20150317_001111	Attack #{SkillAtkAdd}#{nl}{#339999}{ol}[Frozen]{/}{/} Duration 4 seconds
 SKILL_20150317_001112	Create Ice Wall with a chance of freezing the enemy.
 SKILL_20150317_001113	Ice Wall #{CaptionRatio}# {nl}{#339999}{ol}[Ice]{/}{/}{nl} Duration 3 seconds {nl} Ice Wall Duration 10 seconds
 SKILL_20150317_001114	Ice Blast
-SKILL_20150317_001115	{#993399}{ol}[Magic] - [Ice]{/}{/}{nl}Blast frozen enemies in front.
+SKILL_20150317_001115	{#993399}{ol}[Magic] - [Ice]{/}{/}{nl} Inflict damage by exploding a frozen target.
 SKILL_20150317_001116	{#339999}{ol}[Ice]{/}{/}Target to Attack #{SkillAtkAdd}#
 SKILL_20150317_001117	Create wall of ice and defend against attacks.
 SKILL_20150317_001118	Enemies attacked {#339999}{ol}[Ice]{/}{/}Rate 50% {nl} {#339999}{ol}[Ice]{/}{/}Duration 3 seconds
 SKILL_20150317_001119	Gust
-SKILL_20150317_001120	Create strong wind to push away enemy. Inflict damage on frozen enemies or nearby Ice Wall.
+SKILL_20150317_001120	Creates a strong wind pushing the target away. Inflicts damage to a target who is frozen or nearby an Ice Wall.
```

Comment:
"Launches/creates a gust of wind that pushes the target/targets(is it AoE?) away. Inflicts damage on enemies that are frozen, or positioned near an Ice Wall"?